### ISSUE TITLE
Undefined index when using eval and debugging

### ISSUE THREAD

**Issue by kkFelix** [smarty-php/smarty#266]
Minimal example with the current 3.1.30 release:

smarty.php:

```
<?php
error_reporting(E_ALL);
ini_set('display_errors',1);

require_once 'vendor/autoload.php';

$s = new Smarty();
$s->setErrorReporting(E_ALL);
$s->setDebugging(true);
$s->setTemplateDir(dirname(__FILE__))
  ->setCompileDir(dirname(__FILE__))
  ->assign('test', '{$smarty.now}')
  ->display('smarty.tpl');

```

smarty.tpl:

```
{eval var=$test}
```

Output:

```
1470667559
Notice: Undefined index: start_time in vendor/smarty/smarty/libs/sysplugins/smarty_internal_debug.php on line 147

Notice: Undefined index: start_template_time in vendor/smarty/smarty/libs/sysplugins/smarty_internal_debug.php on line 73
```

**Comment by uwetews** [comment]
The fix is now in the master branch

**Comment by guckstift** [comment]
Sorry, this didn't fix it. Line 73 in `smarty_internal_debug.php`:
> Notice: Undefined index: start_template_time

**Comment by wullix** [comment]
still not resolved on 3.1.34? I have again this bug
